docs/user/environment-variables.rst

```diff
@@ -1,36 +1,51 @@
 Environment Variables
 =====================
 
-Read the Docs supports environment variables when building your project.
-There are two types of them:
+Read the Docs supports two types of environment variables in project builds:
 
-* default environment variables
-* user-defined environment variables
-
-Both are merged together during the build process and exposed to all the commands executed when building the documentation.
+* `Default environment variables`_
+* `User-defined environment variables`_
 
+Both are merged together during the build process and are exposed to all of the executed commands.
 
 Default environment variables
 -----------------------------
 
-Read the Docs builder sets the following environment variables when building your documentation:
+Read the Docs builders set the following environment variables automatically for each documentation build:
 
-.. csv-table:: Environment Variables
-   :header: Environment variable, Description, Example value
-   :widths: 15, 10, 30
+.. envvar:: READTHEDOCS
 
-   ``READTHEDOCS``, Whether the build is running inside Read the Docs, ``True``
-   ``READTHEDOCS_VERSION``, The Read the Docs slug of the version which is being built, ``latest``
-   ``READTHEDOCS_VERSION_NAME``, Corresponding version name as displayed in Read the Docs' version switch menu, ``stable``
-   ``READTHEDOCS_VERSION_TYPE``, Type of the event triggering the build, ``branch`` | ``tag`` | ``external`` (for :doc:`pull request builds </pull-requests>`) | ``unknown``
-   ``READTHEDOCS_PROJECT``, The Read the Docs' slug of the project which is being built, ``my-example-project``
-   ``READTHEDOCS_LANGUAGE``, The Read the Docs' language slug of the project which is being built, ``en``
+    Whether the build is running inside Read the Docs.
 
-.. note::
+    :Default: ``True``
+
+.. envvar:: READTHEDOCS_VERSION
+
+    The :term:`slug` of the version being built, such as ``latest``, ``stable``,
+    or a branch name like ``feature-1234``. For `pull request builds </pull-requests>`,
```

Comment:
```suggestion
    or a branch name like ``feature-1234``. For `pull request builds </pull-requests>`_,
```

Reply:
```suggestion
    or a branch name like ``feature-1234``. For :doc:`pull request builds </pull-requests>`,
```

Reply:
Not sure which one is the one wanted here.

Reply:
Oops, yeah it was supposed to be `:doc:...` if I had copy pasted correctly.